payments_service/auth/tests/test_views.py

```diff
@@ -0,0 +1,90 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.urlresolvers import reverse
+
+import mock
+from nose.tools import eq_
+from requests.exceptions import HTTPError
+from slumber.exceptions import HttpClientError
+
+from payments_service.base.tests import SolitudeTest
+
+
+class TestSignIn(SolitudeTest):
```

Comment:
I'm always two minds about it but in this case it does separate some concerns nicely. Fixed.